background/transport.js

```diff
@@ -261,6 +261,11 @@ export default class Transport {
 
   fetchAccountsForCache (origin) {
     return this.transport.execute('parity_getDappsAddresses', origin)
+      .then(null, () => {
```

Comment:
Yes, I thought that `catch` doesn't return promise, so you can only use it at the very end, but it seems that that those two are equivalent, will update this.